src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    // Main methods
+    operate(callback, value) {
+      callback(this, value);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    // Operations
+    add(obj, value) {
+      obj.result += value;
+    },
```

Comment:
```suggestion
    add(value) {
      this.result += value;
    },
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**

Reply:
doesn't work, i tried
![image](https://github.com/mate-academy/js_advanced_calculator/assets/34485522/46ae9eb4-9784-4c1f-b04c-fb8b0a60f54a)

Reply:
this is probably because the implementation of operate function is also wrong